web-console/src/lib/components/streaming/inspection/SQLValueDisplay.tsx

```diff
@@ -0,0 +1,14 @@
+import { getValueFormatter, SQLValueJS } from '$lib/functions/ddl'
+import { useMemo } from 'react'
+import { ColumnType } from 'src/lib/services/manager'
+
+import { Typography } from '@mui/material'
+
+export const SQLValueDisplay = ({ value, type }: { value: SQLValueJS; type: ColumnType }) => {
```

Comment:
why not create a type called TypedSqlValue which has both value and type?

Reply:
A good idea, but will require quite some refactoring to do consistently which is not a priority